### ISSUE TITLE
Take new Snapshots

### ISSUE THREAD

**Issue by Eskaaa** [morewings/cra-template-quickstart-redux#9]
Hello,

I'm trying to figure out what to do to take new snapshots.
I just added a new component to familiarize myself with React and Redux.
In order to be able to push the current status to my git repository, I have to take a new snapshot.

```shell

    SyntaxError: ~/my-project/src/components/Counter/Counter.spec.js: Unexpected token (30:6)

      28 |     /* We can only use enzyme `mount`, no `shallow`, since we are using React hooks, which `shallow` doesn't support */
      29 |     const wrapper = mount(
    > 30 |       <Provider store={store}>
         |       ^
      31 |         <Counter />
      32 |       </Provider>
      33 |     );

      at Parser.raise (node_modules/@babel/parser/src/parser/location.js:41:63)
      at Parser.unexpected (node_modules/@babel/parser/src/parser/util.js:150:16)
      at Parser.parseExprAtom (node_modules/@babel/parser/src/parser/expression.js:1123:20)
      at Parser.parseExprSubscripts (node_modules/@babel/parser/src/parser/expression.js:529:23)
      at Parser.parseMaybeUnary (node_modules/@babel/parser/src/parser/expression.js:509:21)
      at Parser.parseExprOps (node_modules/@babel/parser/src/parser/expression.js:279:23)
      at Parser.parseMaybeConditional (node_modules/@babel/parser/src/parser/expression.js:234:23)
      at Parser.parseMaybeAssign (node_modules/@babel/parser/src/parser/expression.js:185:21)
      at Parser.parseExprListItem (node_modules/@babel/parser/src/parser/expression.js:2077:18)
      at Parser.parseCallExpressionArguments (node_modules/@babel/parser/src/parser/expression.js:817:14)

```

Perhaps it would also be helpful for other beginners to describe this in more detail.

**Comment by morewings** [comment]
Can you share the full code of your test and component?

> Perhaps it would also be helpful for other beginners to describe this in more detail.
What kind of info would be useful for you?

**Comment by Eskaaa** [comment]
Many thanks for the answer.
No source code is required for this explanation.

I resolved my problem by adding this line to the scripts section in my package.json
I needed to add the path to setupTests.js, because otherwise it ignores the React Adapter for me.

```json
    "snapshot": "jest --updateSnapshot --setupFiles ./src/setupTests.js"
```

It would have been helpful to me if the README had told me what to do for a GIT push with own changes. I wanted to leave the provided configuration as original as possible and use it as this template was provided.
